ieeeuoft/hackathon-template
# IEEE 213: load user data into team detail page

## Overview

- Resolves #IEEE-213
- loads user data into teamInfoTable on teamDetailPage
- i don't know how to write redux tests but will add them later


## Unit Tests Created

- none


## Steps to QA

- /teams/{team_id}
- make sure you enter a team id, not a team code. they are different and the page expects an id. check django admin site to find team ids
